src/components/login/login.jsx

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import LoginForm from './loginForm/LoginForm';
+import logo from '../../assets/images/logo/logo.png';
+import './login.scss';
+import { useSelector } from 'react-redux';
+import { Redirect } from 'react-router-dom';
+
+const Login = () => {
+	const auth = useSelector((state) => state.login.isAuth);
+	if (auth) return <Redirect to='/profile' />;
+	return <div className='login'>
+		<img src={logo} alt="" className='logo' />
+		<LoginForm />
+	</div>;
+};
+
+export default Login;
```

Comment:
move the file to the "src/pages" folder

Reply:
done.
https://github.com/mtalstykh-exadel/team4_frontend/blob/27f79d12b2ded8e5b31bd2b44a4a2d0ffd8adfa8/src/pages/login.jsx#L1-L17